.circleci/config.yml

```diff
@@ -4,10 +4,28 @@ orbs:
   terraform: circleci/terraform@2.1.0
 
 executors:
-  linux-machine:
+  cross-builder:
+    docker:
+      - image: quay.io/influxdb/cross-builder:go1.17.1-41d944d749b2c130119eff7efaaa9883ccffe5a7
```

Comment:
Does this image target need to get updated in conjunction with anything else? Presumably with any update to the go version right? It might be worth it to put a comment about this anywhere we pin the go version

Reply:
Yes, the hope is that this will eventually be the one place to change when bumping Go version, but for now there are a few places to coordinate. I'll add a comment.